lib/board.rb

```diff
@@ -0,0 +1,16 @@
+# this is a class for setting the board for game
+class Board
+  attr_accessor :board
```

Comment:
- [ ] You should use  attr_readers or attr_writers. Only using attr_accessors it allows writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Reply:
for the reviewers: unfortunately, this pull request got accidentally merged when we were merging the 2nd milestone. Based on the review we have added validation for empty inputs. We have put all the "puts" in the main file.